src/forms/formHelpers.js

```diff
@@ -347,6 +347,15 @@ const IntervalColumnHeadings = function ({ type }) {
 
 /** 
  * @todo description
+ * 
+ * @param {Object} props
+ * @param {number || string} props.value - Valid client value
+ * @param {string} props.name - For HTML name property
+ * @param {string} props.className - HTML class names
+ * @param {*} [props.otherData] - Sent back to `store()`
+ * @param {function} props.format - Given `value`. Must return what you want shown in the number field. Not sure how to jsdoc this.
```

Comment:
With the `@callback` tag I think. http://usejsdoc.org/tags-callback.html

Reply:
Thank you for the info, that's a great piece of knowledge! I think for now, given time constraints, I'll put a todo in there about it and leave it for a later date.